### PR TITLE
chore: Using trampoline strategy for releases

### PR DESCRIPTION
**Pull request by ddixit14**

### CODE REVIEW

File: .kokoro/trampoline.sh [hunk 1]
```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+# Copyright 2018 Google Inc.
```
Comment (JoeWang1127) [inline .kokoro/trampoline.sh:2]: Do we need to update the year in Copyright?

File: .kokoro/stage.sh [hunk 2]
```diff
@@ -28,17 +28,8 @@ MAVEN_SETTINGS_FILE=$(realpath .)/settings.xml
 setup_environment_secrets
 create_settings_xml_file $MAVEN_SETTINGS_FILE
 
-# run unit tests
-./mvnw verify --show-version --batch-mode
```
Comment (meltsufin) [inline .kokoro/stage.sh:32]: Why remove the testing?

**Review by meltsufin — CHANGES_REQUESTED** [review]
I just realized we already use trampoline in google-cloud-java. It looks like this is matching that config.

**Review by meltsufin — APPROVED** [review]

File: .kokoro/stage.sh (hunk 2 above)
Comment (ddixit14) [reply .kokoro/stage.sh:32]: Added it.

**Pull request merged**